src/python_osw_validation/__init__.py

```diff
@@ -107,22 +107,34 @@ def validate(self, max_errors=20) -> ValidationResult:
             unmatched = node_ids_edges_u - node_ids
             is_valid = len(unmatched) == 0
             if not is_valid:
+                unmatched_list = list(unmatched)
+                displayed_unmatched = ', '.join(map(str, unmatched_list[:20]))
                 self.errors.append(
-                    f"All _u_id's in edges should be part of _id's mentioned in nodes, _u_id's not in nodes are: {unmatched}")
+                    f"All _u_id's in edges should be part of _id's mentioned in nodes. "
+                    f"Showing 20 out of {len(unmatched)} unmatched _u_id's: {displayed_unmatched}"
+                )
 
             # Do all node references in _v_id exist in nodes?
             unmatched = node_ids_edges_v - node_ids
             is_valid = len(unmatched) == 0
             if not is_valid:
+                unmatched_list = list(unmatched)
+                displayed_unmatched = ', '.join(map(str, unmatched_list[:20]))
                 self.errors.append(
-                    f"All _v_id's in edges should be part of _id's mentioned in nodes, _v_id's not in nodes are: {unmatched}")
+                    f"All _v_id's in edges should be part of _id's mentioned in nodes. "
```

Comment:
What happens when the `unmatched_list` has less than 20 elements? The error message will become something like.. `Showing 20 out of 10 unmatched _v_ids`